frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/utilization-card.tsx

```diff
@@ -51,49 +50,51 @@ const UtilizationCard: React.FC = () => {
           <UtilizationDurationDropdown />
         </CardActions>
       </CardHeader>
-      <UtilizationBody>
-        <PrometheusUtilizationItem
-          title={t('ceph-storage-plugin~Used Capacity')}
-          utilizationQuery={UTILIZATION_QUERY[StorageDashboardQuery.CEPH_CAPACITY_USED]}
-          humanizeValue={humanizeBinaryBytes}
-          byteDataType={ByteDataTypes.BinaryBytes}
-          TopConsumerPopover={storagePopover}
-        />
-        <PrometheusMultilineUtilizationItem
-          title={t('ceph-storage-plugin~IOPS')}
-          queries={[
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_IOPS_READ_QUERY],
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_IOPS_WRITE_QUERY],
-          ]}
-          humanizeValue={humanizeIOPS}
-          chartType="stacked-area"
-        />
-        <PrometheusMultilineUtilizationItem
-          title={t('ceph-storage-plugin~Throughput')}
-          queries={[
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_THROUGHPUT_READ_QUERY],
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_THROUGHPUT_WRITE_QUERY],
-          ]}
-          humanizeValue={humanizeDecimalBytesPerSec}
-          chartType="stacked-area"
-        />
-        <PrometheusMultilineUtilizationItem
-          title={t('ceph-storage-plugin~Latency')}
-          queries={[
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_LATENCY_READ_QUERY],
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_LATENCY_WRITE_QUERY],
-          ]}
-          humanizeValue={humanizeLatency}
-          chartType="grouped-line"
-        />
-        <PrometheusUtilizationItem
-          title={t('ceph-storage-plugin~Recovery')}
-          utilizationQuery={
-            UTILIZATION_QUERY[StorageDashboardQuery.UTILIZATION_RECOVERY_RATE_QUERY]
-          }
-          humanizeValue={humanizeDecimalBytesPerSec}
-        />
-      </UtilizationBody>
+      <div className="co-utilization-card__body">
+        <Grid>
```

Comment:
```suggestion
        <Grid className="co-utilization-card__body">
```
WhaT about doing this?

Reply:
Changed.